typespec-tests/tspconfig.yaml

```diff
@@ -10,5 +10,4 @@ options:
     flavor: "Azure"
     dev-options:
       generate-code-model: true
-      support-versioning: true
```

Comment:
I think you need to set this flag in script, for resiliency/servicedriven test -- they are for the advanced case to solve SDK API breaks (I am not sure whether tsp cmd can set the flag under a object, if not we can move this one level above).

Reply:
Yes, it can. Updated.

Reply:
It may be easier to disable it in tspconfig, and only enable on that particular test. Up to you.

Reply:
OK. If we are using basic versioning for default mode, this is better.

Reply:
Yeah, this is in dev-options, so it definitely not a default.

Reply:
Emmm, seems I can't get it to work under an object... Maybe previously I got the test wrong(misconfigured tspconfig.yaml). Let me put it to the root level.

Reply:
moved, renamed to `advanced-versioning` to distinguish between basic versioning.